test/Microsoft.AspNet.Mvc.Core.Test/Controllers/DefaultControllerFactoryTest.cs

```diff
@@ -101,15 +102,23 @@ public void CreateController_SetsBindingContext()
                 ControllerTypeInfo = typeof(ControllerWithAttributes).GetTypeInfo()
             };
             var bindingContext = new ActionBindingContext();
-
+            var bindingContextAccessor = new ActionBindingContextAccessor
```

Comment:
It used to work because `GetServices()` set up the `IActionBindingContextAccessor` instance. Why was that removed?